pages/docs/reference/javascript-dce.md

```diff
@@ -30,33 +31,47 @@ To keep certain declarations from elimination, add the `dceTask` block to your G
 list the declarations as arguments of the `keep` function. An argument must be the declaration's fully qualified name
 with the module name as a prefix: `moduleName.dot.separated.package.name.declarationName`
 
-<div class="multi-language-sample" data-lang="groovy">
-<div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
+> Unless specified otherwise, the names of functions and modules can be [mangled](js-to-kotlin-interop.html#jsname-annotation) in the generated JavaScript code. To keep such functions from elimination, use the mangled names in the `keep` arguments as they appear in the generated JavaScript code.
+{:.note}
+
+
+<div class="sample" markdown="1" mode="groovy" theme="idea">
 
 ```groovy
-kotlin.target.browser {
-    dceTask {
-        keep 'myKotlinJSModule.org.example.getName', 'myKotlinJSModule.org.example.User'
+kotlin {
+    js {
+        browser {
+            dceTask {
+                keep("myKotlinJSModule.org.example.getName", "myKotlinJSModule.org.example.User" )
+            }
+            binaries.executable()
+        }
     }
 }
 ```
-
-</div>
 </div>
 
-<div class="multi-language-sample" data-lang="kotlin">
-<div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
+If you want to keep a whole package or module from elimination, you can use its fully qualified name as it appears in the generated JavaScript code.
+
+> Keeping whole packages or modules from elimination can prevent DCE from removing many declarations that otherwise would have been removed. Because of this, it is preferable to select individual declarations which should be excluded from DCE one by one.
```

Comment:
```suggestion
> Keeping whole packages or modules from elimination can prevent DCE from removing many unused declarations. Because of this, it is preferable to select individual declarations which should be excluded from DCE one by one.
```
If we shorten the sentence like this, will it keep the meaning?

Reply:
Yup, that still sounds good to me!